include/doocsServerTestHelper.h

```diff
@@ -254,7 +253,13 @@ std::vector<TYPE> DoocsServerTestHelper::doocsGetArray(const std::string& name)
   ed.set(&iiii);
   // obtain values
   p->lock();
+
+  // Try to get the data with parameters for a spectrum
   p->get(&ad, &ed, &res);
+  if(res.error() == eq_errors::not_implemeted) {
```

Comment:
not without changing all tests to use a dedicated getSpectrum() instead of providing this generic getArray(). 

in the error case, the type of res is STRING, unfortunately